server/lib/PLS/Server.pm

```diff
@@ -209,6 +214,9 @@ sub send_message
 
     return if (not blessed($message) or not $message->isa('PLS::Server::Message'));
     my $json   = $message->serialize();
+
+    JJ::jjlog('S_', $json);
```

Comment:
What is S_? Why is the underscore after the S here but before the C above?

Reply:
Server.
The underscore helps in reading the log files:
S_: {... message from the server }
_C: {... message from the client }